src/panels/video-player/VideoPlayer.tsx

```diff
@@ -87,9 +90,9 @@ export const VideoPlayer = (props: VideoPlayerPropsFromParent) => {
       // Load in VideoPlayer component
       setVideoData(
         props.appKitTMDataSource.videoData({
-          entityId,
-          componentName,
-          kvsStreamName,
+          entityId: entityId as string,
+          componentName: componentName as string,
+          kvsStreamName: kvsStreamName as string,
```

Comment:
Can `videoData` handle these arguments if they're arrays or will it throw an error? Since `getDisplayOptionValue` returns a `UrlQueryValue`, it's possible it can also be one of these types `string[] | number[] | boolean[]`. But the `VideoDataProps` type expects only strings.

Some options for removing the type assertions from this file.
1. Update `getDisplayOptionValue` to only ever return a string.
2. Update videoData to support `UrlQueryValue`, and update the type for the displayOptions state like so (the type can be extracted instead of being defined inline if needed)
```tsx
const [displayOptions, setDisplayOptions] = useState<{
    entityId: UrlQueryValue;
    componentName: UrlQueryValue;
    kvsStreamName: UrlQueryValue;
    search: UrlQueryValue;
    startTime: UrlQueryValue;
    endTime: UrlQueryValue;
  }>({
    entityId: '',
    componentName: '',
    kvsStreamName: '',
    search: search,
    startTime: '',
    endTime: '',
  });
```

Reply:
I believe `UrlQueryValue` can only be an array if the variable has a value and the string `tempVarVal` would be returned. The only way that `UrlQueryValue` is returned is if some label is in the URL that doesn't match an existing variable in a dashboard. This is possible through the TwinMaker plugin, but we don't support arrays.

Reply:
Updated `getDisplayOptionValue` to return a string. If the type cast gets wonky then there's something wrong with the dashboard setup anyway.